test/tests/misc/block_boundary_material_check/bc_check.i

```diff
@@ -0,0 +1,31 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+[]
+
+[Variables]
+  [u]
```

Comment:
Are we sure we want to drop the dots and slashes? I really think we should wait for the python support to catch up. How does this work with Peacock?

Reply:
I will fix these up.